src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        String[] colors = {"red", "blue", "green", "yellow", "purple"};
+        Random random = new Random();

```

Comment:
Please make random class variable in order not to create new object each time method is called